pydantic_ai_slim/pydantic_ai/models/__init__.py

```diff
@@ -1019,18 +1019,20 @@ def infer_model(  # noqa: C901
     if model_kind in (
         'openai',
         'azure',
+        'cerebras',
         'deepseek',
         'fireworks',
         'github',
         'grok',
         'heroku',
+        'litellm',
         'moonshotai',
-        'ollama',
+        'openai',
+        'openai-chat',
+        'openrouter',
+        'perplexity',
         'together',
         'vercel',
-        'litellm',
-        'nebius',
-        'ovhcloud',
```

Comment:
We lost a bunch of stuff here! Please review the entire PR line by line before passing it to me for review :)